README.md

```diff
@@ -1,2 +1,11 @@
 # QA-merge-demo
 LUT School assignment
+
+### Installing 
```

Comment:
For the sake of clarity, there could be information where the installing command should be run ;)